### PR TITLE
[9.0][MIG] website_sale_suggest_create_account: Upgrade to v9 

### PR DESCRIPTION
**Pull request by lasley**
This is the other half of #89 - website_sale_suggest_create_account, which I submitted out of order 😆

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Same comments about the commit than in #124

File: website_sale_suggest_create_account/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+   :target: https://runbot.odoo-community.org/runbot/113/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+e-commerce/issues>`_. In case of trouble, please check there if your issue has
+already been reported. If you spotted it first, help us smashing it by
+providing a detailed and welcomed feedback `here <https://github.com/OCA/
+e-commerce/issues/new?body=module:%20
```
Comment (pedrobaeza) [inline website_sale_suggest_create_account/README.rst]: Update this to latest version

File: website_sale_suggest_create_account/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Suggest to create user account when buying",
+    "summary": "Suggest users to create an account when buying in the website",
+    "version": "9.0.1.0.0",
+    "category": "Website",
+    "website": "http://www.antiun.com",
+    "author": "Antiun Ingeniería S.L., "
```
Comment (pedrobaeza) [inline website_sale_suggest_create_account/__openerp__.py]: Please switch this to the new company `Tecnativa`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: website_sale_suggest_create_account/__openerp__.py (hunk 2 above)
Comment (lasley) [reply website_sale_suggest_create_account/__openerp__.py]: Copyrights too, or just this author key?

I'll also try to keep this in mind and apply the change in other things going forward.

**Comment by lasley** [comment]
@pedrobaeza - Updates made. Because I was rebasing and destroying our conversation anyways, I went ahead and just squashed the fix commit into my original.

I would love to hear if you have a strategy for removing that merge commit, but I can't figure out a way to combine the disparate parents without it.

Side note - now that I'm looking at Github, comments aren't being collapsed after being handled anymore. Double edged sword because our conversations aren't erroneously destroyed on rebasing, but now verifying what was done just became incredibly more difficult 😦

**Comment by pedrobaeza** [comment]
Yeah, the comments not dissapearing seems a bit odd. Maybe they will dissapear when approving changes...

Please remove also the merge commit and you can also squash all the Yajo's commits in one.

**Comment by lasley** [comment]
@pedrobaeza - How do I remove the merge commit? These were disparate parents because the module was not on 9.0. Is there some sort of wizardry that I don't know about here? It's part of the migration instructions as well, so we should update if you've got something fancy up your sleeve.

Will do on the squashing.

**Comment by pedrobaeza** [comment]
Other times that I made the method where the module is not in branch, I didn't have a merge commit, so I'm not sure why it appears. Anyway, please try the new experimental mode, that also works when the module doesn't exist, and thus we can see if we switch the instructions to this one.

**Comment by lasley** [comment]
@pedrobaeza - Done

That new process was really difficult, there were conflicts almost every step of the way. I feel like I was doing something wrong, but I repeated it a few times. Can you point me to where this development is happening? I don't see an issue in maintainer-tools?

**Comment by pedrobaeza** [comment]
It happened in the mailing list (some prefer this channel because is more "general audience"): https://odoo-community.org/groups/contributors-15/contributors-44716?mode=thread&date_begin=&date_end=

**Comment by lasley** [comment]
Thanks @pedrobaeza - I'll mess around locally a bit more then bring it to that thread. 

For the moment I reset then did some hilariously ghetto cherry-picks off of the backup from my initial branch which seem to have achieved the intended results.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Merging this one to speed up PR treatment.

**Pull request merged**